### ISSUE TITLE
Error in `bt_getbalance()` for some currencies

### ISSUE THREAD

**Issue by rorynolan** [ropensci/bittrex#9]
For some currencies like BTC and ETH, the `bt_getbalance()` function works fine, but for others, I get e.g. 
```
bt_getbalance("BTG")
#> Error in (function (..., row.names = NULL, check.rows = FALSE, check.names = TRUE,  : 
  arguments imply differing number of rows: 1, 0
```
Perhaps this is an error in the bittrex API itself but you need a more informative error message here.
Thanks!

**Comment by kaneplusplus** [comment]
The problem is that I convert the json payload to a list. When an element is `NULL` and I try to turn it into a data frame it is converted to a zero element vector yielding the error.

It is fixed now.  If you haven't created an address, then the field is now returned as an `NA` in the resulting `data.frame` object.

**Comment by rorynolan** [comment]
Wow, that was quick. Thanks for the package, I'm loving trying it out.

**Comment by kaneplusplus** [comment]
Thanks! 

In case you are interested, I'll mention that I'm currently working on a related package [bittrextools](https://github.com/kaneplusplus/bittrextools) to provide higher level functionality for trading. It is currently under development.

**Comment by rorynolan** [comment]
Just checked out `bittrextools`. Looks like more great functionality. Any reason you're making it its own package rather than just adding those functions to the `bittrex` package?

**Comment by kaneplusplus** [comment]
I wanted the package to be as simple and as stable as possible. There was also some concern when I submitted to rOpenSci that `bittrex` was a package for automated trading (not science). So, the scope of the package was limited to querying the exchange.

However, I build automated trading algorithms and it was clear that the private interface is too low-level. So, I started writing `bittrextools` to fill in the gap. This package will never include strategies, but it should make implementing strategy execution much easier.
